aiger/bv.py

```diff
@@ -481,17 +517,59 @@ def __ge__(self, other):
 
     def rename(self, name):
         """Renames the output of the expression; mostly used internally"""
-        rename_map = {self.name(i): f'{name}[{i}]' for i in range(self.size)}
-        return BV(
-            self.size, (self.variables, self.aig['o', rename_map]), name=name)
+        m = {self.name(i): f'{name}[{i}]' for i in range(self.size)}
+        return BV(self.size, (self.variables, self.aig['o', m]), name=name)
+
+    def __call__(self, args={}, signed=True):
```

Comment:
Avoid using mutable types as default arguments. If it accidently gets updates below then the next time it gets called the default argument won't be the empty dictionary. Very hard to debug.

Recommendation:
```python
def __call__(self, args=None, signed=True):
    if args is None:
        args = {}
```